prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404, Prac_09,
+SilverServiceTaxi class inherits from Taxi class.
+"""
+from prac_09.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Specialised version of a Taxi."""
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness):
```

Comment:
You've implemented the code perfectly, but you need to write a short statement under function!